completions/fio

```diff
@@ -10,17 +10,21 @@ _fio()
             return
             ;;
         --debug)
-            _comp_delimited , -W "process file io mem blktrace verify
-                random parse diskutil job mutex profile time net rate compress
-                steadystate helperthread"
+            local opts=$("$1" --debug=help 2>/dev/null)
+            opts=${opts##*:}
+            _comp_delimited , -W "${opts//,/ }"
             return
             ;;
         --output-format)
             COMPREPLY=($(compgen -W "terse json json+ normal" -- "$cur"))
             return
             ;;
         --terse-version)
-            COMPREPLY=($(compgen -W "2 3" -- "$cur"))
+            COMPREPLY=($(compgen -W "2 3 4" -- "$cur"))
```

Comment:
It seems the latest version supports also `5`. Somehow `5` is not included in `fio --help` with fio 3.29 / Fedore 36, but the man page mentions it. In fact, `fio --terse-version=5` seems to correctly parse while `fio --terse-version=6` fails in the argument-parse phase.

Reply:
`5` added in 25a84598a3c8ea7a6ea0b76205a6bbbe1f905ef0